metricbeat/module/docker/vendor/vendor.json

```diff
@@ -0,0 +1,274 @@
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "p6Ud4Yf1ywWy20YxXF1RU4yhTio=",
+			"path": "github.com/docker/docker/pkg/homedir",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "ofQiK0pEBkGOTcAIBhlmecuk/tA=",
+			"path": "github.com/docker/docker/pkg/idtools",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "MgSwrRRAfCQrrToxPAoMCmbieLE=",
+			"path": "github.com/docker/docker/pkg/ioutils",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "BlFSSK7zUjPzPuxkLmM/0wpvku8=",
+			"path": "github.com/docker/docker/pkg/jsonlog",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "bVriSfB8Dbm5C/FAe2k/Uxlljxc=",
+			"path": "github.com/docker/docker/pkg/jsonmessage",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "ndnAFCfsGC3upNQ6jAEwzxcurww=",
+			"path": "github.com/docker/docker/pkg/longpath",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "oHqQiESBEfIJ+uu1+ue7PHvJ6CI=",
+			"path": "github.com/docker/docker/pkg/pools",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "txf3EORYff4hO6PEvwBm2lyh1MU=",
+			"path": "github.com/docker/docker/pkg/promise",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "FHOOypod1NsTgQffapTFhtZ49+8=",
+			"path": "github.com/docker/docker/pkg/stdcopy",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
+			"checksumSHA1": "sjdNnoPZipse7blbn/DzfxVKpe4=",
+			"path": "github.com/docker/docker/pkg/system",
+			"revision": "f02993b2050240bbe7f2bff51f9be5b91e57b978",
+			"revisionTime": "2017-03-31T09:16:51Z",
+			"version": "17.03.1-beats-branch",
+			"versionExact": "v1.13.1"
+		},
+		{
```

Comment:
why are there two `github.com/ruflin/docker` in this vendor.json? The first one seemed right, where we are trying to use a fork of docker. This one doesnt seem to be right